keyboards/model01/keymaps/numist-skcm/keymap.c

```diff
@@ -0,0 +1,135 @@
+/* Copyright 2018 James Laird-Wah
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include "keymap_steno.h"
+
+enum custom_keycodes {
+    M_FIND = SAFE_RANGE,
+    M_AGAIN,
+    M_UNDO,
+    M_CUT,
+    M_COPY,
+    M_PSTE,
+    DYNAMIC_MACRO_RANGE
+};
+
+#define DYNAMIC_MACRO_SIZE 256
+#define DYNAMIC_MACRO_RANGE SAFE_RANGE
+#include "dynamic_macro.h"
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+    case M_UNDO:
```

Comment:
You can use `LGUI(KC_Z)` in your keymap to do this, no need for a macro here.  

And same with the rest of these.

Reply:
Yeah… this code started out the same as the maps using `use_gui` until I realized that the model 01 only ever gets used with macOS 😅